infrastructure/lambda/template.yaml

```diff
@@ -566,6 +574,13 @@ Resources:
         - AWSXrayWriteOnlyAccess
         - DynamoDBReadPolicy:
             TableName: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
+        - Statement:
```

Comment:
We could see it SAM Connectors could be used to simplify permissions - 

https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/managing-permissions-connectors.html

Reply:
Oh cool! Like:
````
  MyConnector:
    Type: AWS::Serverless::Connector
    Properties:
      Source:
        Id: MyFunction
      Destination:
        Id: MyTable
      Permissions:
        - Write
       ````